easybuild/easyconfigs/q/QTLtools/QTLtools-1.1-intel-2016b.eb

```diff
@@ -0,0 +1,39 @@
+# Easyconfig for QTLtools
+# Author: Caspar van Leeuwen
+# SURFsara, Amsterdam, The Netherlands
+
+easyblock = 'MakeCp'
+
+name = 'QTLtools'
+version = '1.1'
+
+homepage = "https://qtltools.github.io/qtltools/"
+description = """QTLtools is a tool set for molecular QTL discovery and analysis. It allows to go from the raw sequence data to collection of molecular Quantitative Trait Loci (QTLs) in few easy-to-perform steps."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['https://qtltools.github.io/qtltools/binaries/']
+sources = ['QTLtools_%(version)s_source.tar.gz']
+checksums = ['9ee0d69d40e3a1827b8805eb98c1bf8a']
+
+#Overwrite CXX, CXXFLAG, LIB_FLAGS and LIB_FILES build options from the default makefile, as these use hard-coded compiler & library paths
+buildopts = 'CXX="$CXX -std=c++0x" CXXFLAG="$CXXFLAGS" LIB_FLAGS="-lz -lgsl -lmkl -lbz2 -lm -lpthread -lRmath -lhts -lboost_iostreams -lboost_program_options" LIB_FILES=""'
+
+moduleclass = 'bio'
+
+dependencies = [
+	('GSL','2.3'),
+	('bzip2','1.0.6'),
+	('Boost','1.61.0'),
+	('Rmath','3.3.1'),
+	('HTSlib','1.4')
+	]
+
+sanity_check_paths = {
+	'files': [],
+	'dirs': ['bin']
+}
+
+#MakeCp specific:
+files_to_copy = ['bin']
```

Comment:
move this up, above `sanity_check_paths`, not sure if the comment provides much useful information?